impl/pkg/storage/db/bolt/pkarr.go

```diff
@@ -1,32 +1,19 @@
-package storage
+package bolt
 
 import (
-	"github.com/goccy/go-json"
+	"context"
+	"encoding/json"
+
+	"github.com/TBD54566975/did-dht-method/pkg/storage/pkarr"
 )
 
 const (
```

Comment:
should this also move to `impl/pkg/storage/pkarr/pkarr.go`?

alternatively could move it (and the contents of the new `pkarr.go`) to a new files `impl/pkg/storage/models.go`

Reply:
this file is the boltdb-specific pkarr implementation. I will rearrange it a bit be more clear what it's doing.